src/components/DirectoryStateGrid.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import { twMerge } from "tailwind-merge";
+import { Address } from "@yext/pages/components";
+import { formatPhoneNumber, formatPhoneNumberIntl } from 'react-phone-number-input';
+
+
+export const directoryStateGridFields = [
+    "c_addressRegionDisplayName",
+    "dm_directoryParents.name",
+    "dm_directoryParents.slug",
+    "dm_directoryParents.meta",
+    "dm_directoryChildren.name",
+    "dm_directoryChildren.slug",
+    "dm_directoryChildren.dm_directoryChildrenCount"
```

Comment:
Don't use `dm_directoryChildrenCount`. It's wrong and going away. Use `dm_childEntityIds`. Same in others.